pyccel/ast/core.py

```diff
@@ -38,7 +38,6 @@
 # TODO [YG, 12.03.2020]: Rename classes to avoid name clashes in pyccel/ast
 # NOTE: commented-out symbols are never used in Pyccel
```

Comment:
I think that this "NOTE" is no longer relevant. Please remove it

Reply:
Fixed in 908cae3